.travis.yml

```diff
@@ -15,15 +15,18 @@ script:
   - >
     sbt coverage
     "++$TRAVIS_SCALA_VERSION test"
-    "++$TRAVIS_SCALA_VERSION tut"
     "++$TRAVIS_SCALA_VERSION doc"
     "++$TRAVIS_SCALA_VERSION publishLocal"
 
   # Compile example project
   - (cd example; sbt ++$TRAVIS_SCALA_VERSION test)
 
-  # check if there are no changes after `tut` runs
+  # Run tut and check if there are no changes in docs.
+  #
+  # We're running tut only in Scala 2.12 because it fails the compile step in
+  # the presence of comments in 2.13: https://github.com/tpolecat/tut/issues/246
```

Comment:
We should consider switching to mdoc, as their engine seems much less amenable to weird edge cases. I tried to do it a few months ago but I found some blockers; maybe it's worth visiting that again.

Reply:
I agree, although it won't help us here right now because mdoc is not yet available for Scala 2.13.

Reply:
Another good reason to move to mdoc eventually is that is that Rob is deprecating tut:

https://github.com/tpolecat/tut/issues/246#issuecomment-519943218